src/include/mpir_op.h

```diff
@@ -178,7 +193,6 @@ int MPIR_NO_OP_check_dtype(MPI_Datatype);
         }                                                \
     } while (0)                                          \
 
-#define MPIR_PREDEF_OP_COUNT 14
```

Comment:
does removing this line belong to this commit?

Reply:
Good catch. It should be removed in a separate commit. It is unused.